components/theme-toggler.tsx

```
@@ -24,56 +24,61 @@
     );
   }
 
   return (
     <Popover placement="bottom-end" offset={4}>
       <PopoverTrigger>
-        <button 
+        <button
           className="relative flex items-center gap-1.5 px-3 py-1.5 text-sm bg-gradient-to-r from-gray-100 to-gray-200 dark:from-gray-900/95 dark:to-gray-800/95 backdrop-blur-sm text-gray-800 dark:text-white rounded-lg hover:from-gray-200 hover:to-gray-300 dark:hover:from-gray-800/95 dark:hover:to-gray-700/95 transition-all duration-300 border border-gray-100 dark:border-gray-700/50 hover:border-gray-400 dark:hover:border-gray-600/70 group overflow-hidden"
-          aria-label={`Current theme: ${theme || 'loading'}`}
+          aria-label={`Current theme: ${theme || "loading"}`}
         >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)